python/test.py

```diff
@@ -3,17 +3,19 @@
 
 from libsonata import *
 
+
 PATH = os.path.dirname(os.path.realpath(__file__))
 PATH = os.path.join(PATH, '../tests/data')
 
+
 class TestSelection(unittest.TestCase):
     def test_basic(self):
         ranges = [(3, 5), (0, 3)]
         selection = Selection(ranges)
         self.assertTrue(selection)
         self.assertEqual(selection.ranges, ranges)
         self.assertEqual(selection.flat_size, 5)
-        self.assertEqual(selection.flatten(), [3, 4, 0, 1, 2])
+        self.assertEqual(selection.flatten().tolist(), [3, 4, 0, 1, 2])
```

Comment:
assert_array_equal instead ?

Reply:
You mean, `numpy.testing`, right?
`tolist()` in effect checks also the result type (i.e. that it is an `array`, not a `list`).
This time maybe it's a desired effect.

Reply:
Ok, I see